packages/moonstone/TooltipDecorator/TooltipDecorator.js

```diff
@@ -31,6 +31,19 @@ let currentTooltip; // needed to know whether or not we should stop a showing jo
  * @hocconfig
  */
 const defaultConfig = {
+	/**
+	 * The boundary around the screen which the tooltip should never cross, typically involving
+	 * flipping to an alternate orientation or adjusting its offset to remain on screen.
+	 * The default of 24 is derived from a standard 12px screen-keepout size plus the standard
+	 * Spotlight-outset (12px) margin/padding value which keeps elements and text aligned inside a
+	 * [Panel]{@link moonstone/Panels.Panel}.
+	 *
+	 * @type {Number}
+	 * @default 24
+	 * @memberof moonstone/TooltipDecorator.TooltipDecorator.defaultConfig
+	 */
+	screenEdgeKeepout: (12 + 12),
```

Comment:
Is `keepout` a well-known term? Wondering if it will be clear to developers what this means.

Reply:
"Keepout" is a standard television term, dating back to the days of CRT monitors. The "keepout" space was generally agreed upon in all of TV history as a recognized dead-zone. You'll notice the effects of this on old TV shows that only bothered to make their graphics cover the usable space (and not paint in the keepout space). They'll abruptly stop at the screen edge, but not the transmissions' edge.